ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageCvePage.tsx

```diff
@@ -227,7 +276,16 @@ function ImageCvePage() {
                             <Flex alignItems={{ default: 'alignItemsCenter' }}>
                                 <EntityTypeToggleGroup
                                     imageCount={imageData?.imageCount ?? 0}
+                                    deploymentCount={deploymentData?.deploymentCount ?? 0}
                                     entityTabs={imageCveEntities}
+                                    onChange={(entity) => {
+                                        // Ugly type workaround
```

Comment:
I couldn't figure out a way to quickly constrain the props passed to `EntityTypeToggleGroup` when passing a known subset of `EntityTab`, so this check is an easy/safe workaround.

Reply:
hmmm not sure what a better/idiomatic way to do this would be, so i'm good with leaving it as is until we find a better solution.

Reply:
I had tried making the component generic instead of using `EntityTab` directly like
```typescript
function EntityTabToggleGroup<Tab extends EntityTab>(props: EntityTabToggleGroupProps<Tab>) {}
```
hoping that TypeScript would infer the more specific type of `props.entityTabs` to use for the generic when I passed a subset of `EntityTab`, but it gave type errors and the cost/benefit of figuring it out wasn't worth it imo.